tests/test-distributed-war/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+FROM ${test.docker.image}
+ADD ${project.build.finalName}.war $APP_DESTINATION
+
+COPY credential.json /credentials/key.json
```

Comment:
The file credential.json should be replaced with the user credential, which could be generated either from gcloud console https://developers.google.com/identity/protocols/application-default-credentials or with the CLI https://cloud.google.com/sdk/gcloud/reference/iam/service-accounts/keys/create